.github/workflows/ci.yml

```diff
@@ -68,7 +70,6 @@ jobs:
             ${{ runner.os }}-pip-
       - name: Install dependencies
         run: |
-          set -xeu
```

Comment:
had to remove this because it makes no sense in a Windows world. Either way, if any of the two operations break, the rest will fail.